OpenRA.Mods.Common/Widgets/Logic/Editor/TileSelectorLogic.cs

```diff
@@ -50,11 +52,31 @@ public TileSelectorLogic(Widget widget, WorldRenderer worldRenderer)
 
 			tileCategorySelector.Text = categories.First();
 			IntializeTilePreview(widget, worldRenderer, tileset, categories.First());
+
+			var searchTextField = widget.GetOrNull<TextFieldWidget>("SEARCH_TEXTFIELD");
```

Comment:
It would be awesome and IMO much more useful if this could work across *all* categories, not just the current one.  The text field could move above the category dropdown to indicate that it wasn't category specific.